docs/LARGE_TESTS.md

```diff
@@ -0,0 +1,165 @@
+# Large Tests: Building and Testing
+
+This guide gets you started with writing and running large tests on Snap plugins. If your plugin needs updated to the large test framework you can find step by step instructions [here](https://github.com/kjlyon/snap/blob/master/Pluginsync_for_large_tests.md).
```

Comment:
We weren't sure where this type of doc should live. Temporarily it is in my repo. Do you have any ideas of where it should belong?